src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -51,6 +59,9 @@ export default defineComponent({
     },
 
     thumbnail: function () {
+      if (this.thumbnailPreference === 'hidden') {
+        return require('../../assets/img/thumbnail_placeholder.svg')
+      }
```

Comment:
ditto

Reply:
This is how my code looks based on this suggestion. (See comment on line 6)

```js
thumbnail: function () {
      let baseUrl
      if (this.backendPreference === 'invidious') {
        baseUrl = this.currentInvidiousInstance
      } else {
         // We still have to check whether or not thumbnails are hidden here
        return this.thumbnailPreference === 'hidden' ? require('../../assets/img/thumbnail_placeholder.svg')  : this.data.playlistThumbnail
      }

      switch (this.thumbnailPreference) {
        case 'start':
          return `${baseUrl}/vi/${this.firstVideoId}/mq1.jpg`
        case 'middle':
          return `${baseUrl}/vi/${this.firstVideoId}/mq2.jpg`
        case 'end':
          return `${baseUrl}/vi/${this.firstVideoId}/mq3.jpg`
        case 'hidden':
          return require('../../assets/img/thumbnail_placeholder.svg')
        default:
          return `${baseUrl}/vi/${this.firstVideoId}/mqdefault.jpg`
      }
    }
  }
```
I can't think of a way right now which wouldn't require making the same check twice different to my last implementation. Maybe there's a better way but idk right now.

Reply:
Your code is actually correct. The order is
- No thumbnail
- Local API = use `playlistThumbnail`
- IV API